routers/api/v1/api.go

```diff
@@ -7,59 +7,59 @@
 //
 // This documentation describes the Gitea API.
 //
-//     Schemes: http, https
-//     BasePath: /api/v1
-//     Version: {{AppVer | JSEscape | Safe}}
-//     License: MIT http://opensource.org/licenses/MIT
+//	Schemes: http, https
```

Comment:
does swagger generator understand this ... i remember an open issue for go1.19 upstream

Reply:
According to CI perhaps not, getting swagger errors.  
I think this PR should likely remain open regardless, as this is going to cause issues with anyone using Go 1.19

Reply:
Adding this PR here to help track: https://github.com/go-swagger/go-swagger/pull/2808